scripts/96-chromebook.install

```diff
@@ -17,51 +17,49 @@ KERNEL_ITB="kernel.itb"
 
 case "$COMMAND" in
     add)
-        if [[ -f "$BOOT_DIR_ABS/$INITRD" ]]; then
-            kernel="${KERNEL_IMAGE##*/}-${KERNEL_VERSION}.lz4"
-            # Compression method being used by coreboot
-            compression="lz4"
-            rm -f "$BOOT_DIR_ABS/${KERNEL_IMAGE##*/}-${KERNEL_VERSION}.lz4" || true
-            lz4 "$BOOT_DIR_ABS/${KERNEL_IMAGE##*/}-${KERNEL_VERSION}" "$BOOT_DIR_ABS/${KERNEL_IMAGE##*/}-${KERNEL_VERSION}.lz4"
+        kernel="${KERNEL_IMAGE##*/}-${KERNEL_VERSION}.lz4"
```

Comment:
I don't know if it makes sense to remove the check for initrd. is it safe to proceed with updating the kernel in case where an initrd image wasn't built?

Reply:
Ah, that was the goal of the check. Makes sense then, I'll drop this commit from the set.